posthog/api/test/test_event.py

```diff
@@ -686,3 +686,694 @@ def test_filter_events_with_date_format(self):
 
         self.assertEqual(len(response["results"]), 1)
         self.assertEqual([r["event"] for r in response["results"]], ["should_be_included"])
+
+
+# Copy paste of TestEvents testing everything with the live-events flag enabled
+# Eventually we'll remove the above class and only use this one once the flag is
+# gone and we enable the feature for everyone
+class TestLiveEvents(ClickhouseTestMixin, APIBaseTest):
```

Comment:
this shouldn't need a review as it's literally copy paste from the class above, just that all methods got this decorator:

```
@patch("posthoganalytics.feature_enabled", return_value=True)
```

There's a way to do this by creating a decorator that we'd add to all methods on the class above but this is the easiest way and actually cleanest / easiest to debug. These tests helped me find a bunch of bugs!